otx/api/utils/segmentation_utils.py

```diff
@@ -67,7 +75,6 @@ def mask_from_annotation(
         2d numpy array of mask
     """
 
-    labels = sorted(labels)  # type: ignore
```

Comment:
@ashwinvaidya17 @samet-akcay Could you double-check if this can affect to anomalib or geti? Because of this line, we had a problem with label mismatch because it was sorted in order of string in id. (e.g. id(0) -> id(1) -> id(10) -> id(11) -> ...) So we have aligned it in string order sort in all codes. But it seems like users will be confused if model head label order and label order in log is different from label json file. Thus, we'd like to align it in the label order as user input.

Reply:
I will revert it actually to
`sorted(labels, key=lambda x: int(x.id))`, because I will remove sorting in entity
@samet-akcay, please let me know if it will affect anomalib

Reply:
From a quick initial test it does not seem to cause as issue as each dataset item has only one label associated with it. So this should be fine for anomaly tasks.